### PR TITLE
[Beta] Add ARIA labels

### PR DESCRIPTION
**Pull request by shamwela**
This will improve accessibility for screen readers. I've checked with Lighthouse locally.

### CODE REVIEW

File: beta/src/components/Search.tsx [hunk 1]
```diff
@@ -95,13 +95,15 @@ export const Search: React.FC<SearchProps> = ({
...
       <button
+        aria-label="Search"
         type="button"
         className="inline-flex md:hidden items-center text-lg p-1 ml-4 lg:ml-6"
         onClick={onOpen}>
         <IconSearch className="align-middle" />
       </button>
 
       <button
+        aria-label="Search"
```
Comment (gaearon) [inline beta/src/components/Search.tsx:106]: This one shouldn't be necessary because there's text inside?

**Comment by harish-sethuraman** [comment]
Would you like look into these as well? 
Screenshot 2022-02-14 at 10 23 00 PM
Screenshot 2022-02-14 at 10 23 10 PM

File: beta/src/components/Search.tsx (hunk 1 above)
Comment (shamwela) [reply beta/src/components/Search.tsx]: I've removed it now. Thanks.

**Comment by shamwela** [comment]
I'll take a look.

**Comment by shamwela** [comment]
I've solved ARIA label issues. I'll solve other accessibility issues in another pull request.

**Review by harish-sethuraman — APPROVED** [review]
LGTM

**Pull request merged**